packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useMappingsOperations.ts

```diff
@@ -61,14 +59,14 @@ const fetchExtractionStatus = async (
         const logs = await extractionClient.getExtractionLogs(accessToken, jobId);
         const filteredLogs = logs.filter((log) => log.message !== null);
         const extractionMessageData = filteredLogs.map((filteredLog) =>
-          (
-            {
-              date: filteredLog.dateTime,
-              category: filteredLog.category,
-              level: filteredLog.level,
-              message: String(filteredLog.message),
-            }
-          ));
+        (
```

Comment:
I think it would be better to move the `({` to the line above. It should be easier to keep the indentation that way.